docs/UserGuide.md

```diff
@@ -144,18 +155,80 @@ Shows a list of all flashcards currently created.
+- Keywords do not need to match exact word. Example, the keyword `Wha` will pick up questions containing `What` as a word.
+
+</div>
+
+1. Use the `list` command to first list all the flashcards.
+
+	![FindQuestionStep1](./images/FindQuestionStep1.png)
+	
+2. Enter the find command followed by `q/` and the keyword in the question of the flashcards you want to find. For example, if you want to find flashcards with the keyword `CS2103T` in their question in the displayed list, you can enter `find q/CS2103T`.
+
+	![FindQuestionStep2](./images/FindQuestionStep2.png)
+	
+3. Press enter and QuickCache will display all the flashcards with the keyword `CS2103T` in their question.
+
+	![FindQuestionStep3](./images/FindQuestionStep3.png)
+
+#### Finding by tags: `find t/TAG1 t/TAG2 ..`
+	
+<div markdown="block" class="alert alert-info">
+
+:information_source: Tags are case-sensitive.
+
+</div>
+
+1. Use the `list` command to first list all the flashcards.
+
+	![FindTagsStep1](./images/FindTagsStep1.png)
+	
+2. Enter the find command followed by `t/` and the tag of the flashcards you want to find. For example, if you want to find flashcards with the tag `Assembly` in the displayed list, you can enter `find t/Assembly`.
+
+	![FindTagsStep2](./images/FindTagsStep2.png)
+	
+3. Press enter and QuickCache will display all the flashcards with the tag `Assembly`.
+
+	![FindTagsStep3](./images/FindTagsStep3.png)
+	
+#### Finding by tags and keywords: `find t/TAG1 t/TAG2 .. q/KEYWORD1 q/KEYWORD2 ..`
+
+<div markdown="block" class="alert alert-info">
 
-Finds all Flashcards based on their tags and/or question.
+:information_source:<br>
+- Words in spaced keywords will be treated as individual keywords. Example, the keyword `what is` will be treated as two keywords: `what` and `is`.<br>
+- Keywords do not need to match exact word. Example, the keyword `Wha` will pick up questions containing `What` as a word.<br>
+- Tags are case-sensitive.
 
-Format: `find t/TAG1 t/TAG2 .. q/KEYWORD1 q/KEYWORD2 ..`
+</div>
+
+1. Use the `list` command to first list all the flashcards.
 
-* Do not need to use both `t/` and `q/` when finding a flashcard.
-* Tags are case-sensitive.
-* Words in spaced keywords will be treated as individual keywords. Example, the keyword `what is` will be treated as two keywords: `what` and `is`.
-* Keywords do not need to match exact word. Example, the keyword `Wha` will pick up questions containing `What` as a word.
+	![FindQuestionAndTagsStep1](./images/FindQuestionAndTagsStep1.png)
+	
+2. Enter the find command followed by `t/` and the tag of the flashcards you want to find. Proceed to include `q/` and the keyword in the question of the flashcards you want to find. For example, if you want to find flashcards with the tag `TCPforLife` in the displayed list and the keyword `What`, you can enter `find t/TCPforLife q/What`.
```

Comment:
> if you want to find flashcards with the tag `TCPforLife` in the displayed list

I believe the mention of the words "in the displayed list" is confusing as it makes the user think that he can only filter against whatever is displayed currently. I think you can just remove "in the displayed list" to make it less confusing.

> the keyword `What`

I think you can add the phrase "in the question" to the end of the statement to make it less ambiguous to the reader